ml.net/InclusiveCodeReviews.Scoring/Program.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System.Text.RegularExpressions;
+using InclusiveCodeReviews.ConsoleApp;
+using InclusiveCodeReviews.Model;
+
+Console.WriteLine("Ctrl+C to exit...");
+Console.WriteLine();
+
+var githubHandleRegex = new Regex(@"\B@([a-z0-9](?:-(?=[a-z0-9])|[a-z0-9]){0,38}(?<=[a-z0-9]))", RegexOptions.IgnoreCase);
+var backtickRegex = new Regex("`+[^`]+`+", RegexOptions.IgnoreCase);
+var urlRegex = new Regex(@"\b(https?|ftp|file):\/\/[\-A-Za-z0-9+&@#\/%?=~_|!:,.;]*[\-A-Za-z0-9+&@#\/%=~_|]", RegexOptions.IgnoreCase);
+var punctuationRegex = new Regex("(\\.|!|\\?|;|:)+$", RegexOptions.IgnoreCase);
+
+while (true)
+{
+	Console.WriteLine("Enter 1 if you want Line prediction or 2 if you want File prediction:");
+
+	var lineOrFile = Console.ReadLine();
+	if (string.IsNullOrEmpty(lineOrFile))
+		continue;
+	if (lineOrFile == "1")
+	{
+		Console.WriteLine("its 1 for Line Prediction");
+		Console.WriteLine("Enter text");
+		var text = Console.ReadLine();
+		if (string.IsNullOrEmpty(text))
+			continue;
+		var result = GetLinePrediction(text);
+		Console.WriteLine($"IsNegative: {result.Prediction}, Confidence: {result.Score[result.Prediction == "1" ? 1 : 0]}");
+		Console.WriteLine();
+	}
+	else if (lineOrFile == "2")
+	{
+		Console.WriteLine("Enter File name not path"); // e.g. sample.txt, good_sampe.txt in comments/ folder
+		var filePath = Console.ReadLine();
+		if (string.IsNullOrEmpty(filePath))
+			continue;
+		var result = GetFilePrediction(filePath);
+		Console.WriteLine($" if score is towards 0 is positive, if score is towards 1 is negative. File Score is : {result}");
+	}
+	else
+	{
+		Console.WriteLine("Not supported Try again");
+	}
+}
+
+string GetFilePrediction(string fileName)
+{
+	var filePrediction = 0;
+	var numLines = 0;
+	string sampleFilePath = Path.Combine(Path.GetDirectoryName(typeof(ModelBuilder).Assembly.Location), "..", "..", "..", "..", "..", "comments", fileName);
+	IEnumerable<string> lines = File.ReadLines(@sampleFilePath);
+	foreach (var line in lines)
```

Comment:
Could this use `CsvHelper` instead of a plain text file?

https://github.com/jonathanpeppers/inclusive-code-reviews-ml/blob/03116a0096e92534513e6f08a42084cfe1166d51/Maui/MLTrainer/ViewModels/ClassificationViewModel.cs#L71-L75

Reply:
This i will address in follow-up PR, i couldn't understand GithubObject, and few more things around it.